go.mod

```diff
@@ -8,15 +8,14 @@ require (
 	github.com/aws/aws-sdk-go-v2/config v1.28.6
 	github.com/aws/aws-sdk-go-v2/service/marketplacemetering v1.25.7
 	github.com/fluxcd/cli-utils v0.36.0-flux.11
-	github.com/fluxcd/pkg/apis/kustomize v1.8.0
+	github.com/fluxcd/pkg/apis/kustomize v1.8.1-0.20250123112748-c55030369b58
 	github.com/fluxcd/pkg/apis/meta v1.9.0
 	github.com/fluxcd/pkg/kustomize v1.15.0
-	github.com/fluxcd/pkg/runtime v0.51.1
+	github.com/fluxcd/pkg/runtime v0.52.1-0.20250128132758-79a726e45f65
```

Comment:
Need to import the proper tag after https://github.com/fluxcd/pkg/pull/858 is merged.